packages/server/test/support/helpers/e2e.js

```diff
@@ -468,13 +472,18 @@ const e2e = {
     _.defaults(options, {
       browser: 'electron',
       project: e2ePath,
-      timeout: options.exit === false ? 3000000 : 120000,
+      exit: process.env.EXIT === 'false' ? false : null,
```

Comment:
i agree with @chrisbreiding, but since you feel very strongly about this, i would at least handle when `exit: false` **is** passed, you provide a detailed error and specify how the user (developer) can achieve the same result with passing the option via the CLI